spec/mailers/claim_mailer_spec.rb

```diff
@@ -16,4 +16,20 @@
       expect(mail.body.encoded).to match("Your unique reference is #{claim.reference}. You will need this if you contact us about your claim.")
     end
   end
+
+  describe "#approved" do
+    let(:claim) { create(:claim, :submitted, first_name: "Abraham", surname: "Lincoln") }
```

Comment:
Can we put a middle name in here to add coverage to make sure we're excluding it?